packages/@okta/vuepress-site/docs/guides/session-cookie/main/index.md

```diff
@@ -1,142 +1,144 @@
 
+### Response Example
 
-``` http
+```
 HTTP/1.1 302 Moved Temporarily
 Set-Cookie: sid=lGj4FPxaG63Wm89TpJnaDF6; Path=/
 Location: https://your-app.example.com?id_token=S4sx3uixdsalasd&state=Af0ifjslDkj&nonce=n-0S6_WzA2Mj
 ```
 
-The response also includes an [ID Token](/docs/reference/api/oidc/#id-token) that describes the authenticated user and can contain additional claims such as user profile attributes or email.
+The response also includes an [ID token](/docs/reference/api/oidc/#id-token) that describes the authenticated user and can contain additional claims such as user profile attributes or email.
 
 The [Okta Sign-In Widget](/code/javascript/okta_sign-in_widget/) uses this flow. This flow can also be used by Single Page Applications with the [`okta_post_messsage`](/docs/reference/api/oidc/#parameter-details) response type, which doesn't require a browser redirect.
 
-## Retrieving a session cookie by visiting a session redirect link
+## Retrieve a session cookie by visiting a session redirect link
 
-> This pattern is supported, but not encouraged. The [OpenID Connect](#retrieving-a-session-cookie-via-openid-connect-authorization-endpoint) flow described above is the preferred pattern for retrieving a session cookie.
+> **Note** This pattern is supported, but not encouraged. The [OpenID Connect](#retrieve-a-session-cookie-through-the-openid-connect-authorization-endpoint) flow described above is the preferred pattern for retrieving a session cookie.
 
-This scenario is ideal for deployment scenarios where you have implemented both a custom login page and custom landing page for your application. Your web application will solicit and validate the user credentials against your Okta organization by calling the [Authentication API](/docs/reference/api/authn/) to obtain a [session token](/docs/reference/api/authn/#session-token).
+This scenario is ideal for deployment scenarios where you implement both a custom sign-in page and a custom landing page for your application. Your web application solicits and validates the user credentials against your Okta organization by calling the [Authentication API](/docs/reference/api/authn/) to obtain a [session token](/docs/reference/api/authn/#session-token).
 
-The session token along with the URL for your landing page can then be used to complete the following [URI Template](http://tools.ietf.org/html/rfc6570) `https://${yourOktaDomain}/login/sessionCookieRedirect?token=${sessionToken}&redirectUrl=${redirectUrl}` that will retrieve a session cookie for a user's browser when visited.
+The session token along with the URL for your landing page can then be used to complete the following [URI Template](http://tools.ietf.org/html/rfc6570) `https://${yourOktaDomain}/login/sessionCookieRedirect?token=${sessionToken}&redirectUrl=${redirectUrl}` that retrieves a session cookie for a user's browser when visited.
```

Comment:
It might be a good idea to add a note here mentioning the new (backward-compatible) `showSignIn` method.

Reply:
And link to the README.

Reply:
Discussed with Jad and he said that showSignIn isn't relevant for the session redirect link scenario.